<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReadRecipe</class>
 <widget class="QWidget" name="ReadRecipe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>930</width>
    <height>767</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Read Recipe</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Filter by Ingredients:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QFrame" name="LeftPanel">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="ingredient_filters_layout">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>184</width>
             <height>595</height>
            </rect>
           </property>
           <widget class="QWidget" name="verticalLayoutWidget">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>771</width>
              <height>511</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="sizeConstraint">
              <enum>QLayout::SetNoConstraint</enum>
             </property>
             <item>
              <layout class="QVBoxLayout" name="ingredient_filters_layout_2">
               <property name="sizeConstraint">
                <enum>QLayout::SetNoConstraint</enum>
               </property>
              </layout>
             </item>
            </layout>
           </widget>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="remove_filter_button">
          <property name="text">
           <string>Remove Filters</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="filter_button">
          <property name="text">
           <string>Apply Filter</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="filter_dropdown">
          <item>
           <property name="text">
            <string>Contains selected ingredients</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Doesn't need other ingredients</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTextBrowser" name="recipe_details_browser">
         <property name="openExternalLinks">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="recipe_list"/>
       </item>
       <item>
        <widget class="QPushButton" name="refresh_button">
         <property name="text">
          <string>Refresh</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
